RetailSuiteAccounting/UserInterface/00a_FiscalYear.md

```diff
@@ -0,0 +1,64 @@
+# Fiscal year
+
+*Accounting*
+
+![Select fiscal year](../../Assets/Screenshots/RetailSuiteAccounting/SelectFiscalYear.png "[Select fiscal year]")
+
+When opening the *Accounting* module for the first time after logging in or after reloading the system, the fiscal year window is displayed. Select the appropriate values to display the corresponding period. For detailed information, see [Select fiscal year](../Operation/01_SelectFiscalYear.md).
+
+The following fields are displayed:
+
+- *Fiscal Year*  
+  Click the drop-down list to select the fiscal year.
+
+- *Period (DD.MM.YYYY) to (DD.MM.YYYY)*  
+  The period fields are filled in automatically and correspond to the selected fiscal year. These fields are read-only.
+
+- *Month*  
+  Click the drop-down list to select the month. The following options are available:  
+
+    - **0 - Balances carried forward**  
+    Period used to post open items and balances carried forward from the previous fiscal year.
+    - **1 - 12 - (calendar month)**  
+    Regular postings are recorded in the corresponding calendar month depending of order data.    
+    - **13 - 15 - Subsequent postings**  
+    Periods used to make postings retrospectively.  
+    - **16 - 25 - Closing postings**  
+    Periods used to make postings at year end.
+    - **99 - General month**  
+    Postings made in this period are not posted in month 99, but are automatically recorded in the corresponding calendar month depending on the posting date.
+
+
+- *Journal*  
+  Enter the journal number. There are two main types of journals preconfigured in the system:
+
+  - General journals (1-12)  
+  They are paired with the fiscal year months, regardless of whether the fiscal year coincides with the calendar year or not. This means that all transactions posted in the first month of the fiscal year will be found in journal 1, all transactions posted in the second month of the fiscal year in the journal 2, and so on. Further journals can be created if necessary. To do so, simply enter a new journal number in the *Journal* field.
+
+  - Special journals (90-99)  
+  They are specialized lists of transactions records preconfigured in the system. The special journals settings can be edited and extended in the BALANCES CARRIED FORWARD tab in the Settings menu entry, see [Balance carried forward](./02h_BalanceCarriedForward.md).
```

Comment:
```suggestion
  Click the [LIST] button next to the field to display the list of journals and select a journal. Alternatively, you can enter the journal number.    
  There are two main types of journals preconfigured in the system:
  - General journals (1-12)  
  The journals 1 to 12 are paired with the fiscal year months, regardless of whether the fiscal year coincides with the calendar year or not. This means that all transactions posted in the first month of the fiscal year will be found in journal 1, all transactions posted in the second month of the fiscal year in the journal 2, and so on.
  - Special journals (90-99)  
  The journals 90 to 99 are specialized lists of transactions records preconfigured in the system. The special journals settings can be edited and extended in the BALANCES CARRIED FORWARD tab in the Settings menu entry, see [Balance carried forward](./02h_BalanceCarriedForward.md).
```
Ein Journal wird erst dann angelegt (bzw. gespeichert), wenn ich in dem entsprechenden Journal eine Buchung habe. Daher ist der Teil mit dem Anlegen etwas verwirrend. Da würde ich hier auch gar nicht drauf eingehen.